### PR TITLE
SC-2294: RedisGui

### PR DESCRIPTION
**Pull request by alexanderM91**
- Ticket: https://spryker.atlassian.net/browse/SC-2994
- RG: https://release.spryker.com/release-groups/view/2626

-----------------------------

### Improvements

- Introduced GUI to access Redis databases and perform some basic operations using RedisCommander.

### CODE REVIEW

File: generator/src/templates/service/redis-commander/default/redis-commander.yml.twig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+  {{ serviceName }}:
+    image: rediscommander/redis-commander:latest
+    networks:
+        - services
+        - private
+    restart: always
+    environment:
+        - REDIS_HOST="{{ serviceData['data']['host'] | default('key_value_store') }}"
```
Comment (sprymiker) [inline generator/src/templates/service/redis-commander/default/redis-commander.yml.twig:8]: Let's use REDIS_HOSTS with kv storage and session and give the possibility to ADD redis services into the list.

Comment (sprymiker) [reply generator/src/templates/service/redis-commander/default/redis-commander.yml.twig:8]: So 2 servers are always there. Custom hosts are only for adding

Comment (alexanderM91) [reply generator/src/templates/service/redis-commander/default/redis-commander.yml.twig:8]: sure

File: generator/src/templates/service/redis-commander/default/redis-commander.yml.twig [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+        - REDIS_PORT="{{ serviceData['data']['port'] | default(6379) }}"
+        - REDIS_DB="{{ serviceData['data']['db'] | default(1) }}"
+    depends_on:
+        - key_value_store
```
Comment (sprymiker) [inline generator/src/templates/service/redis-commander/default/redis-commander.yml.twig:12]: - session

File: generator/src/templates/service/redis-commander/default/redis-commander.yml.twig (hunk 1 above)
Comment (alexanderM91) [reply generator/src/templates/service/redis-commander/default/redis-commander.yml.twig]: I think for this case we can avoid hardcoded (default) hosts and services.
In the deploy.*.yml, a developer can set a different amount of hosts.
e.g in the case of the multi-store concept it’s could be more than one host, with different db number.
    hosts:
       -  hosts1:db1
       -  hosts1:db{N}
       -  host2:db2
   servers:
      - server1
      - server2

If a server doesn’t exist, redis commander, throws an exception.

Comment (sprymiker) [reply generator/src/templates/service/redis-commander/default/redis-commander.yml.twig]: We decided to gather all hosts/databases from deploy.yml and put as hosts for rediscommander.

**Pull request merged**